packages/core/onLongPress/component.ts

```diff
@@ -12,7 +12,7 @@ export const OnLongPress = /* #__PURE__ */ defineComponent<OnLongPressProps>({
   props: ['as', 'options'] as unknown as undefined,
   emits: ['trigger'],
   setup(props, { slots, emit }) {
-    const target = ref()
+    const target = deepRef()
```

Comment:
should this be a `shallowRef` too? since its a ref to an element afaict

Reply:
Yes. I was using a simple regexp to replace those simple ones to shallow, I didn't go through these details (requires context) in this pr. Definitely something for the future.